kubernetes/client/models/v1_affinity.py

```diff
@@ -27,12 +27,12 @@ class V1Affinity(object):
 
     """
     Attributes:
-      openapi_types (dict): The key is attribute name
+      openapi_affinity_types (dict): The key is attribute name
```

Comment:
These files are generated from openapi-generator. Please make the change in the upstream generator.